apisix/plugins/redirect.lua

```diff
@@ -79,7 +96,21 @@ end
 
 
 function _M.check_schema(conf)
-    return core.schema.check(schema, conf)
+    local ok, err = core.schema.check(schema, conf)
+    if not ok then
+        return false, err
+    end
+
+    if conf.regex_uri and #conf.regex_uri > 0 then
+        local _, _, err = re_sub("/fake_uri", conf.regex_uri[1],
+                conf.regex_uri[2], "jo")
+        if err then
+            return false, "invalid regex_uri(" .. conf.regex_uri[1] ..
+                    ", " .. conf.regex_uri[2] .. "): " .. err
```

Comment:
You can use `string.format` to make the error message composition clearer.

Reply:
ok